types/superstruct/index.d.ts

```diff
@@ -0,0 +1,101 @@
+		/** `instance` structs validate that an object is an instance of a particular class, using the built-in instanceof operator. */
+		instance(object: any): Kind;
+		/**
+		 * `interface` structs validate that a value has a set of properties on it, but it does not assert anything about unspecified properties.
+		 * This allows you to assert that a particular set of functionality exists without a strict equality check for properties.
+		 */
+		interface(properties: Types): Kind;
+		/** `intersection` structs validate that a value matches all of many structs. Their arguments are any other validate struct schema. */
+		intersection(structs: Type[]): Kind;
+		/** `lazy` structs accepts a function that will return a struct. They are useful to create recursive structs. */
+		lazy(structs: () => Kind): Kind;
+		/** `list` structs will validate that all of the elements in an array match a specific type. The elements's schema can be any valid value for a struct—string, array, object or function. */
+		list(elements: Type[]): Kind;
+		/** `literal` structs enforces that a value matches an exact, pre-defined value, using the `===` operator. */
+		literal(value: any): Kind;
+		/**
+		 * `object` structs will validate that each of the properties in an object match a specific type.
+		 * The properties's schemas can be any valid value for a struct—string, array, object or function.
+		 */
+		object(properties: Types): Kind;
+		/** `optional` structs validate that a value matches a specific kind of struct, or that it is `undefined`. */
+		optional(type: Type): Kind;
+		/**
+		 * `partial` structs are similar to `object` structs, but they only require that the specified properties exist, and they don't care about other properties on the object.
+		 * They differ from `interface` structs in that they only return the specified properties.
+		 */
+		partial(properties: Types): Kind;
+		/** `scalar` structs are the lowest-level type of struct. They validate that a single scalar value matches a type, denoted by a type string. */
+		scalar(value: string): Kind;
+		/** `tuple` structs validate that a value is a specific array tuple of values. */
+		tuple(values: Type[]): Kind;
+		/** `union` structs validate that a value matches at least one of many structs. Their arguments are any other validate struct schema. */
+		union(schemas: Type[]): Kind;
+	}
+
+	interface Struct {
+		(data: object): any;
+		/** @throws {StructError} */
+		assert(value: object): any;
+		test(value: object): boolean;
+		validate(value: object): [StructError]|[undefined, any];
+	}
+
+	interface StructError extends TypeError {
+		data: any;
+		path: string[];
+		value: any;
+		reason: any;
+		type: string;
+		errors: StructError[];
+	}
+
+	function superstruct(config?: Config): struct;
+
+	type Type = any;
+
+	interface Types {
+		[type: string]: Type;
+	}
+}
+
+declare function superstruct(): void;
+
+export = superstruct;
```

Comment:
There is no `export =` in its source code https://github.com/ianstormtaylor/superstruct/blob/master/src/index.js#L19 , you change just use `export function ...`

Reply:
Just committed the changes